sqlite3_to_mysql/sqlite_utils.py

```diff
@@ -41,7 +41,11 @@ def unicase_compare(string_1: str, string_2: str) -> int:
 def convert_date(value) -> date:
     """Handle SQLite date conversion."""
     try:
-        return date.fromisoformat(value.decode())
+        decoded_value: str = value.decode()
+        if " " in decoded_value:
+            decoded_value = decoded_value.split(" ")[0]
```

Comment:
This smells to me like you're trying to mange a datetime into a date. Not sure I like it.

Reply:
Indeed. I'm sorry that I forgot to remove this piece and reverted that code. It was a bad and legacy database I tried to convert.